spark_log_parser/extractor.py

```diff
@@ -13,7 +13,7 @@
 
 class ExtractThresholds(BaseModel):
     entries = 100
-    size = 5000000000
+    size = 20000000000
```

Comment:
Remove this change, you can pass a new size in as a parameter to the Extractor class.

Reply:
Set 20G limit to default in cli.py